### PR TITLE
Add arin.net

### LINKED ISSUE #6710: Add ARIN
### Site name

ARIN

### Site URL

https://www.arin.net

### Documentation URL

https://www.arin.net/reference/materials/security/twofactor/

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] FIDO2/U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 sites on [Similarweb](https://www.similarweb.com/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by person1234-ux**
resolves #6710 
I wasn't sure what category this website belong it so I just put in under the other category.

### CODE REVIEW

**Comment by phallobst** [comment]
This SVG logo is a bit smaller in file size:
arin net

**Comment by person1234-ux** [comment]
> This SVG logo is a bit smaller in file size: arin net

That just includes one letter from the logo as apposed to the whole logo

**Comment by phallobst** [comment]
It's what they use for their favicon as well, which is our go-to logo variant if the original is too wide to still look good when downscaled to a width of 32 pixels.

**Comment by person1234-ux** [comment]
ok that makes a lot of sense

File: entries/a/arin.net.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Arin": {
```
Comment (phallobst) [inline entries/a/arin.net.json]:
```suggestion
  "ARIN": {
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**